pkgs/http2/lib/transport.dart

```diff
@@ -73,7 +73,7 @@ abstract class TransportConnection {
   Future finish();
 
   /// Terminates this connection forcefully.
-  Future terminate([int? errorCode]);
+  Future terminate([int? errorCode, String? message]);
```

Comment:
Since this class is not `final`, changing the signature of this method is probably breaking. I don't know how important that is.

Reply:
Filed https://github.com/bmw-tech/dart_apitool/issues/207 - I do believe this is a breaking change, so this requires a version rev to `3.0.0`. We should use that opportunity to do some changes like revving the minimum SDK version.